src/index.ts

```diff
@@ -295,10 +295,24 @@ app.get(
 
 // Status endpoint, which returns a JSON of the last import time per endpoint.
 // This uses our internal event sourcing to determine the last imports.
+// Additionally, it returns a list of alerts if any of the imports are too old.
 app.get('/status', async (req: Request, res: Response, next: NextFunction) => {
     try {
+        const yellowAgeHours = 26
+        const redAgeHours = 30
```

Comment:
Do we even retry the job within that time range? 4 hours difference sounds pretty short.

Reply:
increased